config/cmake/config/VXLIntrospectionConfig.cmake

```diff
@@ -61,7 +61,8 @@ macro(PERFORM_CMAKE_TEST PLFM_TEST_FILE TEST)
 
     if(CMAKE_CXX_STANDARD)
       set(TRY_COMP_CXX_STANDARD
-        -DCMAKE_CXX_STANDARD:STRING=${CMAKE_CXX_STANDARD})
+        -DCMAKE_CXX_STANDARD:STRING=${CMAKE_CXX_STANDARD}
+        -DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=${CMAKE_POSITION_INDEPENDENT_CODE})
```

Comment:
CMake automatically propagates this value into try_compile calls.  It does not need to be done explicitly.  I don't think this can make a (good) difference.